html/register/register.html

```diff
@@ -49,7 +49,7 @@ <h1>Register</h1>
                         </div>
```

Comment:
Do we need that here?

Reply:
Yes, Else user will have no way back if they misclick or if they sign up for a account they can press login

Reply:
good jobb!

Reply:
Will do!